### PR TITLE
basic UI setup for dark theme

### PR DESCRIPTION
**Pull request by paul4702**
Hi team, 

If you want to activate the dark mode  

got to 

src\lib\components\header\sub-components\SidebarLinksGroup.svelte , line 276

and uncomment the switch to switch between dark and light theme.

I have cross checked with the current UI of dev and my changes. So that there  shouldn't be any diff. 

If you find any please leave a comment will get on it immediately.

### CODE REVIEW

File: src/lib/page-components/oyster/dashboard/OysterDashboardPage.svelte [hunk 1]
```diff
@@ -99,8 +99,8 @@
 </script>
 
 <div class="w-full">
-	<h2 class="text-3xl font-medium leading-[-2px] text-[#030115]">
-		Hello, {$connected ? shortenText($walletStore.address, 6, 6) : 'Fishy'}
+	<h2 class="text-3xl font-medium leading-[-2px] text-grey-800">
+		Hello, {shortenText($walletStore.address, 6, 6)}
```
Comment (souvikmishra) [inline src/lib/page-components/oyster/dashboard/OysterDashboardPage.svelte:103]: is there a reason why we removed the `$connected` check?

File: src/lib/components/inputs/AmountInputWithMaxButton.svelte [hunk 2]
```diff
@@ -23,9 +23,8 @@
 			<AmountInput
 				bind:value={inputAmountString}
 				onChange={handleUpdatedAmount}
-				styleClass="input input-ghost input-primary text-3xl placeholder:text-[#030115]/[.2]"
-				disabled={!handleUpdatedAmount || !$connected || disabled}
-				{onlyInteger}
+				styleClass="input input-ghost input-primary text-3xl placeholder:text-grey-800/[.2]"
+				disabled={!handleUpdatedAmount || !$connected}
```
Comment (souvikmishra) [inline src/lib/components/inputs/AmountInputWithMaxButton.svelte:27]: is there a reason why we removed `|| disabled` and `{onlyInteger}`

File: src/lib/page-components/oyster/dashboard/OysterDashboardPage.svelte (hunk 1 above)
Comment (paul4702) [reply src/lib/page-components/oyster/dashboard/OysterDashboardPage.svelte:103]: image


I have no idea, How this line was missed. I am updating it

**Comment by souvikmishra** [comment]
also, it feels like some of the classes that we have replaced the hardcoded with are not in both light and dark configs, maybe check once

**Comment by paul4702** [comment]
> also, it seems like some of the classes that we have replaced the hardcoded colors with are not present in both light and dark configs, maybe check once @paul4702

@souvikmishra  The colors we have changed you can find them under the variable names ex: for grey-800 :  --grey-800 & you can find --grey-800 assigened to grey-800 under theme extended colors: PFA screenshot 

 I have changed the most used 6 colors 


image

image


image

File: src/lib/components/header/sub-components/SidebarLinksGroup.svelte [hunk 3]
```diff
@@ -134,11 +134,13 @@
 				: staticImages.ecosystemIcon
 		}
 	];
+
+	$: isDarkModeActive = localStorage?.getItem('theme') === 'light' ? false : true;
```
Comment (vishwajeetraj11) [inline src/lib/components/header/sub-components/SidebarLinksGroup.svelte:138]: this check can be improved.
```$: isLightModeActive = localStorage?.getItem('theme') === 'light'```
or (use dark mode constant)
```$: isDarkModeActive = localStorage?.getItem('theme') === 'dark'```

**Review by vishwajeetraj11 — CHANGES_REQUESTED** [review]

File: src/lib/components/header/sub-components/SidebarLinksGroup.svelte (hunk 3 above)
Comment (paul4702) [reply src/lib/components/header/sub-components/SidebarLinksGroup.svelte]: updated the condition

**Review by vishwajeetraj11 — APPROVED** [review]

**Review by nayanmarlin — CHANGES_REQUESTED** [review]

File: src/lib/components/header/sub-components/SidebarLinksGroup.svelte [hunk 4]
```diff
@@ -271,6 +273,28 @@
 				<MenuItem {...item} />
 			{/each}
 		</ul>
+		<!----------------------------------------------------------------------------------->
+		<!-- Uncomment the following code to enable theme switcher -->
```
[the remark this answers is not in the document]
Comment (paul4702) [reply src/lib/components/header/sub-components/SidebarLinksGroup.svelte:277]: yes, This is a switch so, we want the code to be merged in steps and if somebody wants to test the dark theme they can uncomment it and check how it is looking as it we are pushing it to dev

File: src/lib/page-components/dashboard/MarlinHubDashboard.svelte [hunk 5]
```diff
@@ -114,10 +114,10 @@
 	</p>
 	<div class="mt-6 grid grid-cols-2 gap-4 pb-8">
 		{#each cardsList as card}
-			<div class="flex h-60 w-full flex-col justify-between rounded-2xl bg-white p-6">
+			<div class="flex h-60 w-full flex-col justify-between rounded-2xl bg-secondary-content p-6">
 				<div class="flex flex-col gap-3">
-					<div class="flex items-center gap-2 text-2xl font-medium text-[#030115]">
-						<img src={card.logo} alt={card.logoAlt} />{card.title}
+					<div class="flex items-center gap-2 text-2xl font-medium text-grey-800">
+						<img src={card.logo} alt={card.logoAlt} class="dark:bg-white" />{card.title}
```
[the remark this answers is not in the document]
Comment (paul4702) [reply src/lib/page-components/dashboard/MarlinHubDashboard.svelte:120]: Yes, It was but I am trying out a diff approach to make sure the icons look invert in color. removed it and updated the code

**Review by nayanmarlin — APPROVED** [review]

**Review by souvikmishra — APPROVED** [review]

**Review by souvikmishra — APPROVED** [review]

**Pull request merged**